pkg/config/config.go

```diff
@@ -88,8 +88,12 @@ func LoadConfiguration(cfgFile string, cmd *cobra.Command, printConfig bool) (*C
 		v.SetConfigFile(cfgFile)
 	} else {
 		v.SetConfigName("ct")
-		for _, searchLocation := range configSearchLocations {
-			v.AddConfigPath(searchLocation)
+		if cfgFile, ok := os.LookupEnv("CT_CONFIG_DIR"); ok {
```

Comment:
I get the use case here,  but just checking on implementation. We have `--config` flag now, not `--config-dir`. Why not keep it consistent with `CT_CONFIG` path to file rather than dir?

Reply:
I probalby should have explained that better. `CT_CONFIG_DIR` overrides the default locations where config files are searched. This is not only about the config file for ct but also those for Yamale and yamllint. This will be required when we install ct into a cache location in #58, so config files are still found.

See https://github.com/helm/chart-testing-action/pull/58/files#diff-c141db7e865976e07d678c49af2c6288b4c97d4a0af34ac25905bd46dde8301aR74